packages/sanity/src/core/studio/components/navbar/free-trial/FreeTrial.tsx

```diff
@@ -0,0 +1,71 @@
+import {useCallback, useEffect, useState} from 'react'
+import {Popover} from '@sanity/ui'
+import {PopoverContent} from './PopoverContent'
+import {DialogContent} from './DialogContent'
+import {FreeTrialButton} from './FreeTrialButton'
+import {Wrapper} from './Wrapper'
+import {useFreeTrialContext} from './FreeTrialContext'
+
+interface FreeTrialProps {
+  type: 'desktop' | 'mobile'
+}
+
+export function FreeTrial({type}: FreeTrialProps) {
+  const {data, showDialog, showOnLoad, toggleShowContent} = useFreeTrialContext()
+  //  On mobile, give it some time so the popover doesn't show up until the navbar is open.
+  const [showPopover, setShowPopover] = useState(type !== 'mobile')
+
+  useEffect(() => {
+    setTimeout(() => {
+      setShowPopover(true)
+    }, 300)
+  }, [])
+
+  const closeAndReOpen = useCallback(() => toggleShowContent(true), [toggleShowContent])
+  const toggleDialog = useCallback(() => toggleShowContent(), [toggleShowContent])
```

Comment:
In this case I need it because I need to triger the function with an empty value, that will then be set as false, I updated it now to set always false and make it easier to read.

Reply:
Got it! Sorry for nitpicking!